index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="project-description" content="Portfolio">
+  <meta name="keywords" content="HTML, CSS">
+  <meta name="author" content="Lucy W. Mwangi">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" rel="stylesheet" />
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <!----------nav bar-->
+  <header>
+    <nav>
+      <a>My Logo</a>
+      <img src="Assets/images/Union.png" alt="">
+    </nav>
+  </header>
+
+  <!-------main section------>
+  <section class="main">
+    <div class="intro-text">
+      <h1>I’m Lucy. <br> Glad to see <br> you!</h1>
+      <p class="text-description">I’m a software developer! I can help you build a product, feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</p>
+    </div>
+    <div class="social">
+      <h3>LET'S CONNECT</h3>
+      <ul class="media-icons">
+        <li><a><img src="Assets/images/Twitter.png" alt="twitter"></a></li>
+        <li><a><img src="Assets/images/Linkedin icon.png" alt="linkedin"></a></li>
+        <li><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
+        <li><a><img src="Assets/images/Github.png" alt="github"></a></li>
+        <li><a><img src="Assets/images/peace.png" alt="vector"></a></li>
+      </ul>
+    </div>
+  </section>
+ <!-------work section------>
+ <section class="container-work">
+  <div class="card">
+    <img src="Assets/images/tonic1.svg" alt="" class="image">
+    <div class="left-card1-block">
+      <div class="tonic">
+        <h2>Tonic</h2>
+        <ul class="project-badges">
+          <li><a href="#">Canopy <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
+          <li class="text-2"><a>Back&nbspEnd&nbspDev <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
+          <li class="text-3"><a>2015</a></li>
+        </ul>
+      </div>
+      <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups
+        required.</p>
+      <ul class="language-categories">
+        <li><a href="#">HTML</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">Javascript</a></li>
+      </ul>
+      <button class="submit-btn" type="submit">See Project</button>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement  all `button effects` to your `buttons` like a [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design) 🙏🏼  

![image](https://github.com/lucy-sees/Webforlio/assets/63915024/b2696000-c87c-4037-9a23-33e336a9e615)